packages/server/src/validator.ts

```diff
@@ -0,0 +1,278 @@
+import { JsonRpcError, predefined } from '@hashgraph/json-rpc-relay';
```

Comment:
Good progress. My opinion is that this file way too big, with way too many responsibilities. You can put all const variables in separated `constants.ts` file. Pull out all static methods in separate file as well and keep only one class per file. This way when 3rd person sees the files will know - okay, here are the constants, here we have this class, which does that and so on.. I personally try to aim for higher cohesion.

Reply:
The issue I see here is that this will probably cause circular dependencies and will probably require some level of code/flow refactoring to escape this.